examples/vello_editor/src/main.rs

```diff
@@ -135,7 +136,7 @@ impl ApplicationHandler for SimpleVelloApp<'_> {
                         parley::style::StyleProperty::FontSize(32.0),
                         parley::style::StyleProperty::LineHeight(1.2),
                         parley::style::StyleProperty::FontStack(parley::style::FontStack::Source(
-                            "system-ui",
+                            Cow::Borrowed("system-ui"),
                         )),
```

Comment:
To me, this seems really ugly, but I do like the benefits at other levels...

Reply:
This particular case (constructing styles) could be improved by adding constructors to the style types. Perhaps with with some From/Into implementations so that they can be overloaded.

Reply:
That seems like the way to go yeah.
For now this example should just be 
```Rust
                        StyleProperty::FontStack(FontStack::Single(FontFamily::Generic(
                            GenericFamily::SystemUi,
                        ))),
```
anyway I think.

Reply:
If we really wanted to make it ergonomic we could probably make something like:
```
FontStack::Single(FontFamily::SYSTEM_UI).into()
```

Not convinced ergonomics are that important for parley though. I doubt many people will be interacting with it directly. We have a similar thing with Taffy: we could add style builder but in most cases it actually makes more sense if that sits at the level above.

Reply:
True...